analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -156,6 +156,8 @@ def construct_analyzer_cmd(self, result_handler):
                         compiler_warnings.append('-Wno-' + warning_name)
 
                     continue
+                elif re.match('clang-diagnostic-?$', checker_name):
```

Comment:
This will match only to `clang-diagnostic` and `clang-diagnostic-`. What if I want to disable all checkers which start with `clang-diagnostic-unused` like `clang-diagnostic-unused-variable`, `clang-diagnostic-unused-parameter` etc.?